app/models/host_initiator.rb

```diff
@@ -69,10 +69,10 @@ def delete_host_initiator
 
   # Delete a storage system as a queued task and return the task id. The queue
   # name and the queue zone are derived from the EMS, and a userid is mandatory.
-  def delete_host_initiator_queue(userid)
+  def delete_host_initiator_queue(user)
```

Comment:
@kbrock this feels like a bug in the API we should be passing the string userid down right?

Reply:
@agrare It's been a long time without an update and I want to push the changes, any update here?

Reply:
@kbrock ping
It sounds like the API is passing in the wrong object here

Reply:
It is in the api